alchemy-web/src/content/docs/blog/2025-08-05-alchemy-vite-plugin.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Alchemy Vite Plugin
+excerpt: Bye-bye wrangler.json and .dev.vars, hello alchemy()
```

Comment:
I recommend writing a bit more around the excerpt here. You need something that's a bit more of a hook to reel folks in.